cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -2515,7 +2524,7 @@ impl MachInst for Inst {
                 ));
             } else {
                 let tmp = alloc_tmp(RegClass::I64, types::I32);
-                ret.push(Inst::imm(OperandSize::Size32, value, tmp));
+                ret.push(Inst::imm(OperandSize::Size32, value as u64, tmp));
```

Comment:
Maybe add an assert that the value doesn't overflow the i64 range, in case we'd support i128 in the future? (unless this is already correctly replaced in a subsequent PR)

Reply:
Done!